schema-loader/src/main/java/com/scalar/db/schemaloader/command/CosmosCommand.java

```diff
@@ -47,11 +47,19 @@
   @Option(names = "--no-scaling", description = "Disable auto-scaling for Cosmos DB")
   private Boolean noScaling;
 
+  // For test
   @Option(
-      names = {"--prefix"},
-      description = "Namespace prefix",
+      names = {"--table-metadata-database-prefix"},
+      description = "Table metadata database prefix",
       hidden = true)
-  private String prefix;
+  private String tableMetadataDatabasePrefix;
+
+  // For test
+  @Option(
+      names = {"--coordinator-namespace-prefix"},
+      description = "Coordinator namespace prefix",
+      hidden = true)
+  private String coordinatorNamespacePrefix;
```

Comment:
We can add two parameters for table metadata database prefix and coordinator namespace prefix.